components/bigquery/query/src/query/setup.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Google LLC
```

Comment:
Is setup.py needed?

Reply:
Looks like it wasn't necessary, removed.